subscription/upgrade.md

```diff
@@ -17,28 +17,21 @@ The following section contains information on how to purchase a Docker  subscrip
 
 To purchase a Docker subscription:
 
-1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
+1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account and click **Upgrade**.
 
-2. Open the drop-down menu next to your username in the top-right corner and select **Billing** to go to the **Billing Details** page. This page displays billing information about your existing user account.
+2. Select your subscription and click **Buy Now**.
 
    > **Note**
    >
-   > Ensure you have selected the correct account on the left side of the
-   **Billing Details** page. For example, if you want to purchase a Docker
-   Pro subscription, you must select your personal account.
+   > If you do not have a subscription, navigate to Organizations and click **Create  Organization**. Proceed with the on screen instructions and click **Continue to Payment**.
 
-3. Click **Change plan** and navigate to the pricing page to purchase a subscription and follow the on-screen instructions to purchase.
+3. Complete the on-screen instructions.
 
     > **Note**
     >
-    > You can select from the following options: Pro, Team, or Business
-    subscription. The annual subscription is set by default. Select **Pay
-    Monthly**  to change to a monthly subscription.
-
-4. If you are purchasing a Team subscription, provide the organization details and click **Continue to Org Size**. Review your Team subscription and click **Continue to Payment**.
+    >If you have a Business subscription, 5  is the maximum number of users you can add using a credit card and 40 users for purchase orders. If you want to create a purchase order, update the field for the users and click Contact Sales to complete the contact sales form.
```

Comment:
nitpick: 5 is the minimum number of users. Also an extra space after 5

Reply:
Updated, thanks! @amineck